### PR TITLE
ui updates to sar dashboard table

### PR DESCRIPTION
**Pull request by britt-mo**
### Description
This PR is to update the SAR dashboard table to include Target Populations and update general styling. The due date, last edited, etc data is not expected to be correct in this PR.
Screenshot 2023-10-17 at 10 23 00 AM

### Related ticket(s)

CMDCT-3017

---
### How to test
Login and see that all expected data is there in the SAR table in regular view and mobile view. Also see that the table for the work plan has not been modified.

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/components/pages/Dashboard/MobileDashboardTable.tsx [hunk 1]
```diff
@@ -44,6 +44,12 @@ export const MobileDashboardTable = ({
             </Text>
           </Flex>
         </Box>
+        {reportType === "SAR" && (
+          <Box sx={sx.labelGroup}>
+            <Text sx={sx.label}>Target Populations</Text>
```
Comment (karla-vm) [inline services/ui-src/src/components/pages/Dashboard/MobileDashboardTable.tsx:49]:
```suggestion
            <Text sx={sx.label}>Target populations</Text>
```

**Review by karla-vm — APPROVED** [review]
LGTM! Just one quick update to the headers.

**Review by karla-vm — DISMISSED** [review]

**Review by trancjessica — COMMENTED** [review]
nit, nonblocking!! This may be out of scope for this PR, so happy to approve if that is the case! 

When I tested this locally, it looks like the spacing between `Submission name` and `Target populations`, and before `Status` is a little bit off compared to the mocks 

This is what I see: 
Screenshot 2023-10-17 at 10 42 39 AM

and this is the spacing of the columns on the mocks:
Screenshot 2023-10-17 at 10 44 16 AM

**Comment by britt-mo** [comment]
@trancjessica I talked to design and they said that they plan on addressing design edits after the pen test. But I did agree that the spacing looked a little weird so I added some styling to make it look a little better: 

Screenshot 2023-10-17 at 12 14 54 PM

**Comment by trancjessica** [comment]
> @trancjessica I talked to design and they said that they plan on addressing design edits after the pen test. But I did agree that the spacing looked a little weird so I added some styling to make it look a little better:
> 
> Screenshot 2023-10-17 at 12 14 54 PM

ty for updating it even though we didn't have to!!

**Review by karla-vm — APPROVED** [review]

**Review by trancjessica — APPROVED** [review]
lgtm!!

**Pull request merged**